docs/management/index.md

```diff
@@ -1233,6 +1234,83 @@ DELETE /api/reset
 rabbitmqctl eval 'rabbit_mgmt_storage:reset_all().'
 ```
 
+## Troubleshooting {#troubleshooting}
+
+### OpenId Discovery endpoint not reachable {#openid-Discovery-endpoint-not-reachable}
+
+#### Steps to reproduce the issue
+
+Open the root url of the management ui in the browser, not authenticated yet. 
+Rather than getting the button "Click here to login" you see the following error message:
+
+```
+OAuth resource [rabbitmq] not available. OpenId Discovery endpoint https://<the_issuer_url>/.well-known/openid-configuration not reachable
+```
+
+#### Troubleshoot the issue
+
+These are the most common reasons this issue ocurrs:
+- The browser cannot physyically open a http connection with the OpenId Discovery endpoint. Copy and paste the url in the error message into the browser and see if you can get a reply.
+- If you cannot get a reply then investigate whether the OpenId Discovery endpoint is running at all and/or whether there is any firewall which blocking the access.
+- If you can get a reply then check the browser's console and see if there is an error similar to this one: 
+
+  `Access to fetch at 'https://<the_issuer_url>>/.well-known/openid-configuration' from origin 
```

Comment:
You can use a code block here too, like the previous error example, by indenting the block:

```
- If you (…):

    ```
    Access to fetch (…)
    ```
```